src/index.js

```diff
@@ -18,7 +18,17 @@ function run(argv, env) {
   const rl = readline.createInterface({ input: cp.stdout });
 
   cp.on("error", (err) => {
-    console.error(err);
+    if (options.cmd === 'cargo' && err.code === 'ENOENT') {
```

Comment:
I ❤️  that you check explicitly for the `cargo` command here since this tool can be used with other commands (e.g., `cat build_log`).

Nit, can you use double quotes to match the style of the rest of this file?